src/components/step-nav.scss

```diff
@@ -10,46 +10,48 @@
 // * 5. Titled Modifier
 
 // TODO : document global dependencies
-// TODO : Add Titled Modifier
+// TODO : further responsive cleanup, especially on titled setp nav
 
 //------------------------------------------------------------//
 
 // * 1. Settings
 
 $StepNav-size: 60px;
 $StepNav-border: 4px;
+$StepNav-paddedWidth: $StepNav-size + ($StepNav-border * 2);
```

Comment:
Yeah that's fair, I updated to $StepNav-spacing to be a little more generic since it should cover both that border width and padding